permission-manager/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: permission-manager
+  namespace: permission-manager
+type: Opaque
+stringData:
+  PORT: "4000" # port where server is exposed
+  CLUSTER_NAME: "${CLUSTER_NAME}" # name of the cluster to use in the generated kubeconfig file
+  CONTROL_PLANE_ADDRESS: "https://${MASTER_IP}:6443" # full address of the control plane to use in the generated kubeconfig file
+  BASIC_AUTH_PASSWORD: $(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 10 ; echo '') # password used by basic auth (username is "admin")
```

Comment:
why not use CIVO: ALPHANUMERIC, this way it will be reflected in the user interface

Reply:
@milindchawre can you fix this ? So that we can merge this ?